ci-operator/config/openshift/external-dns/openshift-external-dns-release-1.0.yaml

```diff
@@ -1,12 +1,3 @@
-base_images:
-  base:
-    name: ubi-minimal
-    namespace: ocp
-    tag: "8"
-  ocp_builder_rhel-8-golang-1.17-openshift-4.10:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-1.17-openshift-4.10
 build_root:
   from_repository: true
 images:
```

Comment:
What about the https://github.com/openshift/release/pull/46164/files#diff-ac72eca2732bbeb5457cda7c9db0719444ae42ea7e514af716e2d420856694a2L14-L18 here? Did you miss that?

```
  from: base
  inputs:
    ocp_builder_rhel-8-golang-1.17-openshift-4.10:
      as:
      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10
```

Reply:
Right, it's a miss.